src/mol-model-formats/structure/property/bonds/index-pair.ts

```diff
@@ -45,27 +47,42 @@ export namespace IndexPairBonds {
     export type Data = {
         pairs: {
             indexA: Column<number>,
-            indexB: Column<number>
+            indexB: Column<number>,
+            id?: Column<number>,
             order?: Column<number>,
+            /**
+             * Useful for bonds in periodic cells. That is, only bonds within the given
+             * distance are added. This allows for bond between periodic image but
+             * avoids unwanted bonds with wrong distances.
+             */
             distance?: Column<number>,
             flag?: Column<BondType.Flag>,
         },
         count: number
     }
 
-    export const DefaultProps = { maxDistance: DefaultBondMaxRadius };
+    export const DefaultProps = {
+        /**
+         * If -1, test using element-based threshold, otherwise distance in Angstrom.
```

Comment:
If *negative*, test using element-based threshold, otherwise distance in Angstrom.

?

Reply:
changed it